config/default.yaml

```diff
@@ -36,3 +37,23 @@ modules:
     # - ./modules/statsd
     # - ./modules/openTSDB
     # - ./modules/influxdb
+
+  whitelistedKeys:
```

Comment:
Uncommenting atleast 1 of the keys should fix that issue. This is similar to having to uncomment atleast 1 collectors.